litsupport/shellcommand.py

```diff
@@ -88,6 +88,7 @@ def parse(commandline):
     and will throw an exception if the commandline uses unsupported features.
     """
     result = ShellCommand()
+    commandline = commandline.replace("\\", "/") if os.name == 'nt' else commandline
```

Comment:
Could you just use the if statement syntax here instead of an if expression. 

```
if os.name == 'nt':
    commandline = commandline.replace("\\", "/")
```

Reply:
Fixed by latest commit.